sljit_src/sljitNativeMIPS_common.c

```diff
@@ -435,27 +443,25 @@ static void get_cpu_features(void)
 		if (flag < 0) {
 			SLJIT_ASSERT(freg_map[TMP_FREG3] == 16);
```

Comment:
There should be no such requirement because this is not a jit code refer to this check. We could simply use f0 or any scratch register.

Reply:
It was meant to document what the magic numbers below meant, but agree was a little harsh